orc8r/gateway/python/magma/ctraced/BUILD.bazel

```diff
@@ -36,6 +36,6 @@ py_library(
         "rpc_servicer.py",
         "trace_manager.py",
     ],
-    visibility = ["//visibility:public"],
-    deps = [],
+    visibility = ["//visibility:private"],
```

Comment:
Just out of curiosity:
Is setting the libraries to private part of the new strategy/philosophy coming with the split of `main.py` and "the rest"?

Reply:
No, usually the <service>_lib is used in tests, but here are no tests. This is just set to make it more clear that this lib is only used here. We decided to use this more in the last couple of days. Will be added more regularly in existing places when touched.